src/workspace-data/Data.js

```diff
@@ -700,6 +700,7 @@ export const WorkspaceData = _.flow(
 
     const loadWdsTypes = useCallback(
       (url, workspaceId) => {
+        setWdsTypes({ status: 'None', state: [] });
```

Comment:
Ok this seemed to trick the page into refreshing when the column name updates without obviously breaking anything but it probably secretly breaks something

Reply:
This will unmount WdsContent and recreate it once the types are reloaded.

https://github.com/DataBiosphere/terra-ui/blob/8f9e8efa327b064095818226e6ee77d481b5b755/src/workspace-data/Data.js#L1484-L1493

When WdsContent (and its child DataTable) are recreated, the child DataTable will load the table data.

Reply:
Is that... good?  Bad?  Neutral?

Reply:
I'd say neutral. It's not _ideal_ since there's some unnecessary component destruction/construction, but that's fairly pervasive in this part of the app and it's difficult to fix without major refactors because of Terra UI's use of `useOnMount`.